src/constants.js

```diff
@@ -18,6 +18,8 @@ export const PCT_BASE = bigNum(1)
 
 export const ZERO_ADDR = '0x0000000000000000000000000000000000000000'
 
+export const HIVE_GARDEN_ADDRESS = '0x8ccbeab14b5ac4a431fffc39f4bec4089020a155'
```

Comment:
I think this should go for each network right? We won´t have one for Polygon but we do have for Rinkeby and xDai